frontend/pages/hosts/details/cards/HostSoftwareLibrary/HostSoftwareLibrary.tsx

```diff
@@ -434,20 +434,30 @@ const HostSoftwareLibrary = ({
   }, []);
 
   const onClickInstallAction = useCallback(
-    async (softwareId: number) => {
+    async (softwareId: number, isScriptPackage = false) => {
       try {
         await hostAPI.installHostSoftwarePackage(id as number, softwareId);
         if (isMountedRef.current) {
           onInstallOrUninstall();
         }
+
+        const message = () => {
+          switch (true) {
```

Comment:
lets talk about this pattern today in sync. I get why we're doing it but i personally think if, elseif, and else statements are just easier to understand at a glance than a switch(true) statement. would be good to get other opinions on this